apps/internal/services/fixture_factory.py

```
@@ -2,32 +2,44 @@
 from datetime import datetime, timedelta
```

Flake8 (7.2.0)

[error] 'datetime.datetime' imported but unused

(F401)